docs/step-attributes.md

```diff
@@ -19,6 +19,7 @@ Here are commonly seen use cases that need to be handled carefully, accounting R
 | Step text in Gherkin language | Attribute in feature class | Argument type | Argument value |
 | ----------------------------- | -------------------------- | ------------- | -------------- |
 | Given I chose 15 as second number | [Given(@"I chose (\d+) as second number")] | int | 15 |
+| Given I chose -15 as second number | [Given(@"I chose (-\d+) as second number")] | int | -15. Unfortunately the sign of the number needs to be known up front and cannot change between Examples.  |
```

Comment:
The statement is not entirely accurate. Sign of the number can change between examples if you use `?` for denoting an optional regex symbol match. e.g. this will match both `15` and `-15`: `[Given(@"I chose (-?\d+) as second number")]` (notice `?` after `-`)
Please update the example so that it clarifies this ability.

Reply:
That's great to know. I'll update the example.